sql/mob_pools.sql

```diff
@@ -7147,7 +7147,7 @@ INSERT INTO `mob_pools` VALUES (7071,'Stirge_Oldton','Stirge_Oldton',46,0x000000
 -- Start of AirSkyBoat section
 
 -- Ouryu Cometh
-INSERT INTO `mob_pools` VALUES (20000,'Ouryu','Ouryu',259,_binary 0x0000620200000000000000000000000000000000,7,4,7,240,100,1040,1,1,0,18,648,32,297,1183,0,0,109,0,0,259,259);
+INSERT INTO `mob_pools` VALUES (20000,'Ouryu','Ouryu',259,_binary 0x0000620200000000000000000000000000000000,7,4,7,240,100,1040,1,1,0,18,648,32,297,1183,0,0,109,0,0,4048,259);
```

Comment:
This skill list ID was used in a prior PR. This should also go into a module. All sql changes for updates will be going into modules soon, I just need too find the the time to do it.

Reply:
I moved the Ouryu change to a new PR dealing with several mobskills, will be easier to modularize several there.